aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -190,8 +190,8 @@ AT_FORALL_SCALAR_TYPES_AND3(Bool, Half, BFloat16, DEFINE_CAST_OP)
 
 Tensor empty_like(
     const Tensor& self,
-    c10::optional<c10::MemoryFormat> optional_memory_format) {
```

Comment:
Wont this change the way code behaves?

Reply:
In fact, it did! But in a non-obvious way, see #28093

Reply:
Most of behaviour covered with `test_memory_format_empty_like` and similar tests.